src/parsers/uniqueTokens.js

```diff
@@ -91,6 +106,83 @@ export const parseAccountUniqueTokens = data => {
     .filter(token => !!token.familyName);
 };
 
+export const parseAccountUniqueTokensPolygon = async data => {
+  let erc721s = data?.data?.results;
+  if (isNil(erc721s)) throw new Error('Invalid data from OpenSea Polygon');
+  erc721s = erc721s
+    .map(({ asset_contract, collection, token_id, metadata, ...asset }) => ({
+      ...pick(metadata, [
+        'animation_url',
+        'description',
+        'external_link',
+        'image_original_url',
+        'image_preview_url',
+        'image_thumbnail_url',
+        'image_url',
+        'name',
+        'traits',
+      ]),
+      asset_contract: pick(asset_contract, [
+        'address',
+        'name',
+        'contract_standard',
+      ]),
+      background: metadata.background_color
+        ? `#${metadata.background_color}`
+        : null,
+      collection: pick(collection, [
+        'description',
+        'discord_url',
+        'external_url',
+        'featured_image_url',
+        'hidden',
+        'image_url',
+        'name',
+        'short_description',
+        'slug',
+        'twitter_username',
+        'wiki_link',
+      ]),
+      currentPrice: asset.sell_orders
+        ? `${
+            Number(asset.sell_orders[0].current_price) / 1000000000000000000
+          } ${asset.sell_orders[0].payment_token_contract.symbol}`
+        : null,
+      familyImage: collection.image_url,
+      familyName:
+        asset_contract.address === ENS_NFT_CONTRACT_ADDRESS
+          ? 'ENS'
+          : collection.name,
+      id: token_id,
+      isSendable: false,
+      lastPrice: asset.last_sale
+        ? Number(asset.last_sale?.total_price / 1000000000000000000) +
+          ` ${asset.last_sale.payment_token?.symbol}`
+        : null,
+      lastPriceUsd: asset.last_sale
+        ? asset.last_sale?.payment_token?.usd_price
+        : null,
+      lastSale: asset.last_sale,
+      network: NetworkTypes.polygon,
+      permalink: asset.permalink,
+      type: AssetTypes.nft,
+      uniqueId: `${NetworkTypes.polygon}_${get(
+        asset_contract,
+        'address'
+      )}_${token_id}`,
+      urlSuffixForAsset: `${get(asset_contract, 'address')}/${token_id}`,
+    }))
+    .filter(token => !!token.familyName && token.familyName !== 'POAP');
+
+  //filter out NFTs that are not on our allow list
+  remove(
+    erc721s,
+    NFT => !polygonAllowList.includes(toLower(NFT.asset_contract.address))
```

Comment:
Not a blocker but for follow up release:
1. we can move this filter logic into line 175 with the existing filter logic
2. in references we can transform `polygonAllowList` into a map of keys to some dummy true value so that we can turn the filter from `O(n*m)` (`n` being # of erc721s and `m` being number of items in the polygon allow list) to just `O(n)`.